src/index.html

```diff
@@ -190,6 +190,9 @@
   <!-- For cookie banner -->
   <script src="https://static.ithaka.org/gdpr-banner/js/banner.js" type="text/javascript"></script>
 
+  <!-- Ethnio Activation Code -->
```

Comment:
So we I know we have added/removed the Ethino survey a couple times on artstor. This is an example how we have done it in the past: https://github.com/ithaka/aiw-ui/pull/1162/files . I think it has to do with the fact that artstor is server side rendered.

Wondering if we should be using the `scriptStore`?

Reply:
The actual requirement is to display Ethnio survey in all the pages, meaning once per session but wherever the user lands first, we should be displaying. Since index.html is the main html file that is being used irrespective of which view is displayed ( correct me if I am wrong ), I added the script in index.html. I looked at the PR you included above, which requires adding ScriptService in all the components individually. Personally I prefer index.html because it comes into scene every time when any view displayed.

Reply:
I think `app.component.ts` is the top level angular component. Wondering if we could load ethnio from the script loader in there instead.

Reply:
Yes, I tried to find whether I can use the script in app component. Since, app component doesn't have a view, I found where exactly the selector 'app-root' is being used in the project. Its inside <body> tag of index.html, we are referencing app-root selector which is tied to app component.